DESCRIPTION

```diff
@@ -41,7 +41,8 @@ Imports:
     data.table,
     rlang,
     fastDummies,
-    checkmate
+    checkmate,
+    reshape2
```

Comment:
Could you use base R reshape instead of incurring a dependency on reshape2?

Reply:
Done